### ISSUE TITLE
Is it possible for tests to check which lines contained the error?

### ISSUE THREAD

**Issue by bmitch** [bmitch/Codor#48]

**Comment by winkbrace** [comment]
Yeah, the line number is the key of the `_errors` array in PHP_CodeSniffer_File. 

If you replace the foreach loop in the `\Codor\Tests\Wrappers\Results::getAllErrorMessages()` method like this:

```php
foreach ($errors as $line => $error) {
    $allErrorMessages[] = "Error at line $line: " . reset($error)[0]['message'];
}
```

The output becomes this:
```
[
  0 => "Error at line 5: Return null value found."
  1 => "Error at line 21: Return null value found."
]
```

**Comment by winkbrace** [comment]
Hmm, I just realised that you're logging the error with the `$stackPtr` of the scope start, not the index of the actual token that has an error. Currently error messages point to the start of functions instead of the actual line that contains the error.

I've updated my last PR to report return null errors with the correct stack pointer.

**Comment by bmitch** [comment]
Thanks, I'll take a look at this in the next few days when I have time. But if you feel like tackling it please go ahead.

**Comment by winkbrace** [comment]
I think you should have a look at it first. The error messages would change with that implementation.
And maybe you want to do something different with the lines.

Anyway, I'm on vacation starting now, so won't be programming (much) the coming week. 
Have a good new year! ;)

**Comment by bmitch** [comment]
OK thanks. You too!
